x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/view/components/integration_tests/form.test.tsx

```diff
@@ -139,33 +140,41 @@ describe('When on the host isolation exceptions entry form', () => {
     it('should select the "global" policy by default', () => {
       expect(
         renderResult
-          .getByTestId('effectedPolicies-select-global')
+          .getByTestId(`${testIdPrefix}-effectedPolicies-global`)
           .classList.contains('euiButtonGroupButton-isSelected')
       ).toBe(true);
     });
 
     it('should show policy as selected when user clicks on it', async () => {
       userEvent.click(renderResult.getByTestId('perPolicy'));
-      await clickOnEffectedPolicy(renderResult);
+      await clickOnEffectedPolicy(renderResult, undefined, testIdPrefix);
```

Comment:
Consider passing arguments as an object, or at least switch positions of atIndex with effectedPoliciesPrefix so you wouldn't have to pass `undefined` :)